src/modules/email/email.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+import { MailDataRequired } from '@sendgrid/mail';
+import { SendgridClient } from './sendgrid-client';
+import { CreateUserDto } from '../../modules/user/dto/create-user.dto';
+import { controller } from 'src/constants/controller';
+import { controller_path } from 'src/constants/controllerPath';
+
+@Injectable()
+export class EmailService {
+  constructor(private readonly sendGridClient: SendgridClient) {}
+
+  async sendConfirmationEmail(
+    recipient: string,
+    user: CreateUserDto,
+  ): Promise<void> {
+    const mail: MailDataRequired = {
+      to: recipient,
+
+      from: 'geri.mujo@softup.co',
```

Comment:
@gerimujo496 Would be better if we defined it in an env variable (e.g. mailConfig.sender) instead of hardcoded